src/transaction.rs

```diff
@@ -42,6 +42,9 @@ const TRANSACTION_CREATE_FIXED_SIZE: usize = WORD_SIZE // Identifier
     + WORD_SIZE // Witnesses size
     + Salt::LEN; // Salt
 
+/// Identification of transaction (also called transaction hash)
+pub type TxId = Bytes32;
```

Comment:
This should be moved to `fuel-types` so other crates can have access to it.

Reply:
`TxId` is more transaction thing, but sure I will move it. On another topic, wdyt about moving `UtxoId` to `fuel-types` it has same feel as `TxId`

Reply:
It's not totally clear to me where the dividing line is between tx related types and fuel-types. Would it be confusing to us down the road if we start moving bits of pieces of tx's into fuel-types?

Reply:
You can probably argue that `TxId` and `UtxoId` are native types and that they belong to `fuel-types`. They can be reexported in `fuel-tx`.
For me personally, i dont really care how I am going to include it, i care about merging this and continuing to next thing :)

Reply:
Seconding what @Voxelot. IMO, it wouldn't hurt to first keep these new types here, _then if need arises_ move them to `fuel-types`.

Reply:
One argument to move it immediately is that there's code duplication already: https://github.com/FuelLabs/fuel-tx/pull/53#discussion_r778720939. Avoiding code duplication is probably a `Good Thing`, but can that also be accomplished by exporting the duplicate function? I guess that would unclean, as it's a library of types and not functions.

Reply:
Removed duplicated part, it is now using std function `u8::from_str_radix`. And had a litlle bug there, that is fixed in last commit.